sdk/trace/benchmark_test.go

```diff
@@ -114,6 +118,67 @@ func BenchmarkSpanWithAttributes_all_2x(b *testing.B) {
 	})
 }
 
+func BenchmarkSpanWithEvents_4(b *testing.B) {
+	traceBenchmark(b, "Benchmark Start With 4 Attributes", func(b *testing.B, t trace.Tracer) {
```

Comment:
```suggestion
	traceBenchmark(b, "Benchmark Start With 4 Events", func(b *testing.B, t trace.Tracer) {
```

Check the other benchmark names.